bazel_ros2_rules/ros2/resources/rmw_isolation/__init__.py

```diff
@@ -1 +0,0 @@
-from .rmw_isolation import isolate_rmw_by_path
```

Comment:
Dropping the `rmw_isolation`  feature entirely. It was brittle, we have `network_isolation` now, and it was making the `bzlmod` migration harder.